File: SofiForce.Models/Models/EntityModels/CarModel.cs
using System;

namespace Models
{
    public class CarModel
    {
        public int CarId { get; set; }
        public int? CarTypeId { get; set; }
        public string CarCode { get; set; }
        public string CarNo { get; set; }
        public string Model { get; set; }
        public int? ManufacturerId { get; set; }
        public int? YearManufactur { get; set; }
        public bool? IsActive { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/CarModel.ts
import { ManufacturerModel } from "./manufacturerModel";

export interface CarModel {
    carId: number;
    carTypeId: number | null;
    carCode: string;
    carNo: string;
    model: string;
    manufacturerId: number | null;
    yearManufactur: number | null;
    isActive: boolean | null;
    color: string;
    icon: string;
    displayOrder: number | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    latitude: number | null;
    longitude: number | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
   
}